### PR TITLE
Bug 1772743: Add olm.skipRange to all CSVs and art.yaml (#61)

### PR DESCRIPTION
**Pull request by fabianvf**
(cherry picked from commit 27d3ed355ca2ef9d6cf59df940ff7ecac27e6591)

### CODE REVIEW

**Review by estroz — APPROVED** [review]
/lgtm

File: deploy/olm-catalog/openshift-template-service-broker-manifests/4.1/openshifttemplateservicebrokeroperator.v4.1.0.clusterserviceversion.yaml
Comment (camilamacedo86) [inline deploy/olm-catalog/openshift-template-service-broker-manifests/4.1/openshifttemplateservicebrokeroperator.v4.1.0.clusterserviceversion.yaml:13]: 👍

File: deploy/olm-catalog/openshift-template-service-broker-manifests/4.2/openshifttemplateservicebrokeroperator.v4.2.0.clusterserviceversion.yaml
Comment (camilamacedo86) [inline deploy/olm-catalog/openshift-template-service-broker-manifests/4.2/openshifttemplateservicebrokeroperator.v4.2.0.clusterserviceversion.yaml:13]: 👍

**Review by camilamacedo86 — APPROVED** [review]
/lgtm /approved

**Comment by stevekuznetsov** [comment]
/hold

Please pass test

**Comment by bparees** [comment]
@fabianvf this PR has been held until it passes tests, it won't be cherry-pick-approved until the hold is removed.

**Comment by jwforres** [comment]
@fabianvf bump, see Ben's previous comment

**Comment by crawford** [comment]
@fabianvf bump.

**Comment by crawford** [comment]
Removing from the patch-manager query while we wait for a fix.
/lgtm cancel

**Comment by jmrodri** [comment]
This PR is blocked by PR #72

**Comment by jmrodri** [comment]
blocked by PR #72 Once that merges we can re-run the tests on this PR.

**Comment by jmrodri** [comment]
/hold cancel
PR #72 has merged which should fix CI.

**Comment by jmrodri** [comment]
/lgtm
/approved
This PR was waiting on a CI fix PR #72 to merge. We should be all good now.

**Review by jmrodri — APPROVED** [review]
/lgtm

**Comment by stevekuznetsov** [comment]
/hold
We have tried to report a status to the commits here 1000 times and can't do it anymore. You will need to push a new commit to get tests to work.

**Comment by stevekuznetsov** [comment]
We've re-tested the commits in this PR so many times (1000+) that GitHub won't let us report test results anymore. Please push fixed commits and re-test then.

/hold

**Comment by jmrodri** [comment]
Closing this PR in favor of PR #100.

**Pull request closed without merging**
